ank/src/cli_commands/dry_run_plan.rs

```diff
@@ -0,0 +1,123 @@
+use std::collections::{BTreeMap, BTreeSet};
+use tabled::Tabled;
+use super::cli_table::CliTable;
+use common::objects::WorkloadInstanceName;
+
+#[derive(Clone, Debug)]
+pub enum DryRunAction {
+    NoChange,
+    Create,
+    Recreate,
+    Delete,
+}
```

Comment:
I think the information of the details column is already covered through the action. Maybe we can remove the `details` column and adjust the `action` strings a little bit because what will be run later is not in the focus, it is more about what was changed in the complete state of Ankaios. We can also leave the unchanged workloads away from the output, because it would be more obvious what was changed when you have a lot of workloads.

My suggestion for the new actions:
```suggestion
#[derive(Clone, Debug)]
pub enum DryRunAction {
    Add,
    Update,
    Delete,
}
```